clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -57,6 +57,7 @@ sealed abstract class DeleteCommand[TI <: TransferIndex](val index: TI)
     extends ClioCommand {
   def key: index.KeyType
   def note: String
+  def forceDelete: Boolean
```

Comment:
Bikeshedding: since the command will already be named something like `delete-xyz`, do we need `--force-delete` instead of just `--force`?

Reply:
Should we update the force update on add as well for consistency?

Reply:
I'd say 👍